src/lib/routes/admin-api/project/project-features.ts

```diff
@@ -297,6 +299,29 @@ export default class ProjectFeaturesController extends Controller {
             ],
         });
 
+        this.route({
+            method: 'post',
+            path: PATH_DEPENDENCIES,
+            handler: this.addFeatureDependency,
+            permission: CREATE_FEATURE,
```

Comment:
decided not to introduce new permissions for now and use existing create feature permission